src/lsst/cmservice/db/enums.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Enum
+
+from ..common.enums import (
+    ErrorAction,
+    ErrorFlavor,
+    ErrorSource,
+    LevelEnum,
+    NodeTypeEnum,
+    ProductStatusEnum,
+    ScriptMethod,
+    StatusEnum,
+    TableEnum,
+    TaskStatusEnum,
+    WmsMethod,
+)
+from .base import Base
+
+SqlTableEnum = Enum(TableEnum, metadata=Base.metadata)
```

Comment:
Readability of subsequent code would be better if these all either consistently had "Enum" suffix, or consistently did not?  (Had to verify when reading below whether these were all the same type of thing.)